packages/perseus/src/widgets/interactive-graphs/mafs-styles.css

```diff
@@ -145,6 +145,11 @@
     );
 }
 
+.MafsView
+    .movable-point__focusable-handle:focus-visible {
+    outline: none;
```

Comment:
Very nice! Can you add a comment explaining that this is here because empty elements still have a focus outline in Firefox?